### ISSUE TITLE
Applying Priorities to run in parallel

### ISSUE THREAD

**Issue by hisankaran** [yigit/android-priority-jobqueue#259]
Hi,

I am trying to get a high priority job running immediately when one low priority job is already running. but, the high priority job runs only after the previous job completes. I am using the default settings for JobManager which minConsumer:0, maxConsumer:5, loadFactor:3.

For example Jobs A(Priority=3), B(Priority=1)

Let's say B is being processed and A is enqueued. So, I was expecting A will run in parallel to B.

Should I do any special configuration for this behaviour? Please help me to understand.

Thanks!

**Comment by yigit** [comment]
You've set minConsumer count to 0 and load factor to 3. The load factor 3 means job manager will not create a second thread unless there are more than 3 jobs waiting. 
Priority is only considered at the time of job selection so if the lower priority job came first and it started running, higher priority job has to wait until a new consumer is available.
You can increase the number of minConsumers or decrease the load factor if you want the second job not to wait for the first one.

**Comment by hisankaran** [comment]
Thanks a lot, Yigit! I will try this.
